src/simulator_core/solver/matrix/matrix.py

```diff
@@ -127,6 +129,7 @@ def get_solution(self, index: int, number_of_unknowns: int) -> list[float]:
         be provided
         :return: list with teh solution
         """
+        #  TODO add checks if the index and number of unknowns are within the range of the solution.
```

Comment:
Willen we niet al deze ToDos wegwerken?

Reply:
Niet perse, als het makkelijk en snel te doen is.